tests/phpunit/includes/test-wc-gateway-amazon-payments-advanced.php

```diff
@@ -115,7 +115,7 @@ public function test_classic_process_payment_succeeds() : void {
 				'amazonCreateCheckoutParams' => wp_json_encode( WC_Mocker_Amazon_Payments_Advanced_API::get_create_checkout_classic_session_config( array( 'test' ) ) ),
 				'amazonEstimatedOrderAmount' => wp_json_encode(
 					array(
-						'amount'       => $order_total,
+						'amount'       => number_format( $order_total, 2 ),
```

Comment:
Cant you use the WC_Amazon_Payments_Advanced::format_amount here ? i imagine the method would fail if the decimals set in woo are not 2

Reply:
The idea here is to force decimals to 2, since estimatedOrderAmount parameter fails if decimals is different than 2. But updated to use our own method, it's a wrapper anyways for number_format, but looks nicer 🙂